src/Popover/Popover.js

```diff
@@ -165,7 +166,7 @@ class Popover extends Component {
                     popperProps={{ ...popperProps, id }}
                     referenceClassName='fd-popover__control'
                     referenceComponent={referenceComponent}
-                    show={this.state.isExpanded && !disabled}
+                    show={typeof this.props.show === 'boolean' ? this.props.show && !disabled : this.state.isExpanded && !disabled}
```

Comment:
```js
show={!disabled && (typeof show === 'boolean' ? show : this.state.isExpanded)}
```
To simplify it a little bit, we could take out the repeated `!disabled` as well as use `show` instead of `this.props.show` since it is already deconstructed.

Reply:
on it